en/PlanView/PlanView.md

```diff
@@ -1,47 +1,48 @@
 # Plan View
 
-![](../../images/plan/PlanView.jpg)
+![](../../images/plan/PlanView.png)
 
-The Plan View is used to plan autonomous missions for you Vehicle. Once the mission is planned and sent to the vehicle, you switch to the [Fly View](../FlyView/FlyView.md) to fly the mission.
+The *Plan View* is used to plan autonomous missions for your Vehicle. Once the mission is planned and sent to the vehicle, you switch to the [*Fly View*](../FlyView/FlyView.md) to fly the mission.
 
-If your Vehicle supports a [GeoFence](PlanGeoFence.md) or [Rally Points](PlanRallyPoints.md) you can also set those up from the Plan View.
+If your Vehicle supports a [GeoFence](PlanGeoFence.md) or [Rally Points](PlanRallyPoints.md) you can also set those up from the *Plan View*.
 
-The image above shows a simple mission which starts with a takeoff, flies through two waypoints and then lands.
+The image above shows a simple mission which starts with a takeoff at the home position (H), flies through three waypoints and then lands on the last waypoint (i.e. waypoint 3).
 
 The steps to creating a missions are:
 
-1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
-4. Change to Fly View and fly your mission
+1. Change to *Plan View*.
+2. Add waypoints or commands to the mission and edit as needed.
+3. Upload the mission to the vehicle.
+4. Change to *Fly View* and fly your mission.
 
 ## Plan Tools
-On the left edge of the screen you will see the Plan Tools. The order of tools from top to bottom is:
+On the left edge of the screen you will see the Plan Tools as marked in red in the image above. 
 
-* Add Commands
-* [Survey](Survey.md)
-* Sync
-* Center map
-* Map Type
-* Zoom In/Out
+ > **Note** **Center map**, **Zoom In**, **Zoom Out** are just map visualisation tools that help user view the *Plan view* map clearly while planning missions (they don't affect the mission commands sent to the vehicle in any way).
 
-### Add Commands
-Click to activate the Add Commands tool. While active, clicking in the map will add new mission commands at the clicked location. The tool will stay active until you click it again.
+
+### Add Waypoints
+Click to activate the **Add Waypoint** tool. While active, clicking in the map will add new mission waypoint at the clicked location. The tool will stay active until you click it again. Once you have added a waypoint, you can select it by clicking on it and then drag it around to change its position.
```

Comment:
This is slightly unwieldy. 
1. "Click to activate the " .... it sounds like there is a missing word before "to". 
2. "Click to activate the **Add Waypoint** tool. While active, clicking in the map will add new mission waypoint at the clicked location. While active, clicking in the map will add new mission waypoint at the clicked location. " - too many instances of the word clicked. Perhaps the last one at least should be "selected"